conf/supervisord-dev.conf

```diff
@@ -0,0 +1,50 @@
+[supervisord]
+nodaemon = true
+
+[program:web]
+command = tox -qqe dev --run-command 'pserve --reload conf/development-app.ini'
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
```

Comment:
This doesn't seem to cause any problems (?) and means that `make dev` stops _much_ faster